fastlane/lanes/build.rb

```diff
@@ -259,7 +259,22 @@ def upload_installable_build(product:)
 
     install_url = "#{INSTALLABLE_BUILD_DOMAIN}/#{upload_path}"
     qr_code_url = "https://chart.googleapis.com/chart?chs=500x500&cht=qr&chl=#{CGI.escape(install_url)}&choe=UTF-8"
-    comment_body = "You can test the #{product} changes on this Pull Request by <a href='#{install_url}'>downloading an installable build (#{filename})</a>, or scanning this QR code:<br><a href='#{install_url}'><img src='#{qr_code_url}' width='250' height='250' /></a>"
+    icon = "<img alt='#{product}' align='top' src='https://raw.githubusercontent.com/buildkite/emojis/main/img-buildkite-64/#{product.downcase}.png' width='20px' />"
+    comment_body = <<~PR_COMMENT
+      <details>
+        <summary>#{icon}📲 You can test these changes on #{product} by <a href='#{install_url}'>downloading <tt>#{filename}</tt></a></summary>
+        <table><tr>
+          <td width='250' rowspan='5'><a href='#{install_url}'><img src='#{qr_code_url}' width='250' height='250' /></a></td>
+          <td colspan='2'>💡 Tip: You can also scan this QR code with your Android phone to download and install the APK directly on it.</td>
+        </tr>
+        <tr><td width='150px'><b>Product</b></td><td><tt>#{product}</tt></td></tr>
```

Comment:
Suggestion (💡): If it were me I would have named the `Product` to `App` instead. This naming will make it easier to distinguish what `WordPress` and `Jetpack` is and why. However, I do understand why you chose `product`, to match the variable naming as well, still `product` as a standalone naming feels a bit off to me. Wdyt?

Reply:
✅  Addressed in bd865c2